lib/core/terraform_config/gvc.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module TerraformConfig
+  class Gvc < Base
+    attr_reader :name, :description, :tags, :domain, :locations, :pull_secrets, :env, :load_balancer
+
+    # rubocop:disable Metrics/ParameterLists
+    def initialize(
+      name:,
+      description: nil,
+      tags: nil,
+      domain: nil,
+      locations: nil,
+      pull_secrets: nil,
+      env: nil,
+      load_balancer: nil
+    )
+      super()
+
+      @name = name
+      @description = description
+      @tags = tags
+      @domain = domain
+      @locations = locations
+      @pull_secrets = pull_secrets
+      @env = env
+      @load_balancer = load_balancer&.transform_keys { |k| k.to_s.underscore.to_sym }
```

Comment:
Small nit: I generally prefer using descriptive variable names, like `key`, instead of single letters when possible.

Reply:
I saw that this was removed in the other PR, so nevermind.